psl/parser-database/src/walkers/relation_field.rs

```diff
@@ -14,6 +14,13 @@ use std::{
 #[derive(Copy, Clone, PartialEq, Debug, Hash)]
 pub struct RelationFieldId(pub(crate) ast::ModelId, pub(crate) ast::FieldId);
 
+impl RelationFieldId {
+    /// Coarsen the relation field identifier to a generic field identifier.
+    pub fn coarsen(self) -> (ast::ModelId, ast::FieldId) {
```

Comment:
Absolutely unimportant, and for learning purposes.

Coarsen as in make it in rougher?

Is this idiomatic in any well-known API? isn´t `(ast::ModelId, ast::FieldId)` the actual `inner` representation of a `RelationFieldId`, is `unwrap` common at all?

Reply:
It's an idiom in PSL: in this case, you have a `Field::refine()` method (https://prisma.github.io/prisma-engines/doc/parser_database/walkers/type.FieldWalker.html#method.refine) that gives you a scalar field or a relation field (it's refined because you have more information), and coarsen goes in the other direction: you have a more specific relation field id, and you go back to a generic field id.

Reply:
Beautiful, thank you!